planet/src/main/java/com/google/planet/servlets/ItineraryServlet.java

```diff
@@ -35,52 +37,33 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.io.BufferedReader;
 
-/** Servlet that handles the game two truths and one lie */
 
 @WebServlet("/generate-itinerary")
 public class ItineraryServlet extends HttpServlet {
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        Query query = new Query("Event").addSort("order", SortDirection.ASCENDING);
-        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-        PreparedQuery results = datastore.prepare(query);
+        BufferedReader br = request.getReader();
+		String eventsJson = "";
 
-        // Create an event called "hotel" with duration of 0
-        String startingAddress = request.getParameter("starting-address");
-        Event start = new Event( "Start", 
-                                startingAddress, 
-                                0);
+		if (br != null){
+			eventsJson = br.readLine();
```

Comment:
nit: looks like the indentation here is off.

Reply:
Not sure why it looks off in github, since it looks fine in my local version...